libs/agora/api-description/src/openapi.yaml

```diff
@@ -18,3 +19,7 @@ tags:
 paths:
   /dataversion:
     $ref: paths/dataversion.yaml
+  /getTeams:
```

Comment:
This endpoint should be named `/teams`: HTTP verbs should not be included.

Reply:
good catch!